ctapipe/instrument/camera.py

```diff
@@ -33,19 +33,35 @@
 }
 
 
+_geometry_cache = {}
```

Comment:
no, it's used later in the guess() method. It's a class attribute to keep the list of previously constructed geometries, which are keyed by an integer hash that combines the x,y vectors and the focal length (which I think is enough to make them unique).  Originally, I had used only the number of pixels, but if we use the hexagonal-to-cartesian camera transforms, the number of pixels stays the same but the pixel ordering changes, so it caused some problems.